bin/yarn

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env ruby
-VENDOR_PATH = File.expand_path('..', __dir__)
```

Comment:
Question:

Not sure about the VENDOR_PATH removal

Reply:
they just renamed the variable